packages/SwingSet/test/test-controller.js

```diff
@@ -66,7 +66,7 @@ async function simpleCall(t) {
   t.deepEqual(data.kernelTable, [vatAdminRoot]);
 
   // vat1:o+1 will map to ko21
-  controller.queueToVatExport('vat1', 'o+1', 'foo', capdata('args'));
+  controller.queueToVatRoot('vat1', 'foo', capdata('args'));
```

Comment:
the preceding `vat1:o+1` comment needs updating, since we aren't talking to `o+1` anymore

Reply:
I could swear I did that...